composer.json

```diff
@@ -8,12 +8,18 @@
             "bamarni/composer-bin-plugin": true
         }
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/hemant-mann/flysystem-dropbox.git"
+        }
```

Comment:
This has been added because the latest version 1.0.5 is not published on Packagist (yet?)

We can reliably get 1.0.5 from the tag on GitHub this way.